projects/zxing-scanner/src/lib/zxing-scanner.component.ts

```diff
@@ -99,16 +99,16 @@ export class ZXingScannerComponent implements OnInit, OnDestroy {
   delayBetweenScanSuccess = 500;
 
   /**
-   * Emits when and if the scanner is autostarted.
+   * Emits when and if the scanner is autoStarted.
    */
   @Output()
-  autostarted: EventEmitter<void>;
```

Comment:
That's a breaking change (public API).. Even though it looks better with the uppercase "S" I suggest to revert it

Reply:
no problem, I'll push a change and revert the variable. Should have thought about that. Thanks!